modules/preparing-microshift-executable.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-microshift-executable"]
+= Preparing the MicroShift executable runtime
+:context: creating-microshift-development-environment
+
+Use the following procedure to prepare the MicroShift executable runtime. 
+
+[NOTE]
+====
+When working with MicroShift based on a pre-release minor version `Y` of {OCP}, the corresponding RPM repository `rhocp-4.$Y-for-rhel-8-$ARCH-rpms` will not be available yet. As a workaround, use the `Y-1` version. 
+====
+
+.Prerequisites 
+
+* You have logged into your development virtual machine using your MicroShift user credentials. 
+* You have configured your MicroShift virtual machine. 
+* You have built MicroShift in your development virtual machine. 
+
+.Procedure
+
+. Enable the necessary repositories by entering the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-4.11-for-rhel-8-$(uname -i)-rpms \
```

Comment:
@ggiguash what's the right version repo to use here? Can we predict the 4.12 repo name and use it now?